models/userDao.js

```diff
@@ -0,0 +1,72 @@
+const { appDataSource } = require("./dataSource");
+
+const getUserByKakaoId = async (kakaoId) => {
+  const user = await appDataSource.query(
+    `SELECT
+            id, 
+            kakao_id,
+            first_name,
+            last_name,
+            birth,
+            mobile_number,
+            email   
+        FROM users
+        WHERE kakao_id =?`,
+    [kakaoId]
+  );
+  return user[0];
+};
+
+const storeKakaoId = async (kakaoId) => {
+  return await appDataSource.query(
+    `
+    INSERT INTO users (
+        kakao_id
+    ) VALUES (?);
+    `,
+    [kakaoId]
+  );
+};
+
+const createUser = async (
```

Comment:
kakao_id가 이미 저장 되어 있는 user 정보를 update하는 것이기 때문에 함수 이름을 다음과 같이 바꿔주면 좋을 것 같아요.
```suggestion
const updateUser = async (
```